Sources/AWSLambdaEvents/APIGateway+V2.swift

```diff
@@ -93,24 +93,21 @@ extension APIGateway.V2 {
 
 extension APIGateway.V2 {
     public struct Response: Codable {
-        public let statusCode: HTTPResponseStatus
-        public let headers: HTTPHeaders?
-        public let multiValueHeaders: HTTPMultiValueHeaders?
-        public let body: String?
-        public let isBase64Encoded: Bool?
-        public let cookies: [String]?
+        public var statusCode: HTTPResponseStatus
+        public var headers: HTTPHeaders?
+        public var body: String?
+        public var isBase64Encoded: Bool?
+        public var cookies: [String]?
 
         public init(
             statusCode: HTTPResponseStatus,
             headers: HTTPHeaders? = nil,
-            multiValueHeaders: HTTPMultiValueHeaders? = nil,
```

Comment:
not needed any longer?

Reply:
It never was supported in the first place. My mistake. Quote from [docu](https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-develop-integrations-lambda.html):

> To customize the response, your Lambda function should return a response with the following format.hv
> 
>  ```json
> {
>     "cookies" : ["cookie1", "cookie2"],
>     "isBase64Encoded": true|false,
>     "statusCode": httpStatusCode,
>     "headers": { "headerName": "headerValue", ... },
>     "body": "Hello from Lambda!"
> }     
>  ```

Reply:
sounds good, need to point out in the release notes